src/internal.c

```diff
@@ -11280,12 +11278,6 @@ int BuildMessage(WOLFSSL* ssl, byte* output, int outSz, const byte* input,
     }
 #endif
 
-    /* catch mistaken sizeOnly parameter */
-    if (sizeOnly && (output || input) ) {
```

Comment:
This code was added to protect against accidental misuse of API resulting in an un encrypted message.  Why is it being removed?